packages/core/src/routes/init.ts

```diff
@@ -15,34 +16,31 @@ import swaggerRoutes from '@/routes/swagger';
 
 import adminUserRoutes from './admin-user';
 import roleRoutes from './role';
-import { AnonymousRouter, AuthedRouter } from './types';
-
-const createRouters = (provider: Provider) => {
```

Comment:
what's the matter with this?

Reply:
See Slack thread [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1651830270878339?thread_ts=1651824287.360339&cid=C02A8G4HVAM)

Reply:
Refactored.

```ts
  for (const router of createRouters(provider)) {
    apisApp.use<unknown, any>(router.routes()).use(router.allowedMethods());
  }
```